cli/waiter/token_post.py

```diff
@@ -11,6 +11,8 @@
     load_json_file
 
 BOOL_STRINGS = TRUE_STRINGS + FALSE_STRINGS
+INT_PARAM_SUFFIXES = ['-failures', '-index', '-instances', '-length', '-level', '-mins', '-secs']
+FLOAT_PARAM_SUFFIXES = ['-factor', '-rate', '-threshold']
```

Comment:
Nitpick: we should have a unit test that verifies that there is no intersection between these lists.

Reply:
I would say it's not necessary :)